### ISSUE TITLE
Docker definition for SQL Server 2019 installation

### ISSUE THREAD

**Issue by pulla2908** [microsoft/mssql-docker#684]
In [docker definition](https://github.com/microsoft/mssql-docker/blob/master/windows/mssql-server-windows-developer/dockerfile) SQL Server 2017 gets downloaded and installed. There are 2 download links for EXE and BOX. 
```` 
# Download Links:
ENV exe "https://go.microsoft.com/fwlink/?linkid=840945"
ENV box "https://go.microsoft.com/fwlink/?linkid=840944"
````
Instead of 2017 I want to have 2019. I wonder how I could obtain proper download links for SQL Server 2019 instead? I was searching for it but I did not find them (neither for 2017 nor 2019).

Any idea where to get this from?

**Comment by amvin87** [comment]
Hi @pulla2908 windows bases SQL containers are yet in preview and hence the docs are not updated for SQL 2019. The link to download the SQL 2019 dev is available here: https://go.microsoft.com/fwlink/?linkid=866662 let me also see if I can get this updated. The delay is because this was in preview. But SQL on Linux container is completely supported for production.

**Comment by pulla2908** [comment]
Hi @amvin87, thx for feedback. Would be very great if you could get this updated. Can you maybe then leave a message here once you updated it?

By the way I tried to sign up for [SQL Server 2019 preview program](https://cloudblogs.microsoft.com/sqlserver/2019/07/01/sql-server-2019-on-windows-containers-now-in-early-adopters-program/) but when I submit the form I get a 500 response. However, do you think it makes sense to sign up and follow the progress there?

**Comment by soroshsabz** [comment]
@amvin87 Why Microsoft does not support SQL Server on Microsoft OS officially? When you think Microsoft support SQL Server 2019 officially support on Windows Server 2019?

Thanks

**Comment by soroshsabz** [comment]
@amvin87 I think it is very weird to Microsoft push its users and developer community to other platform (competitor platform).

I think Windows Server is live platform and Microsoft can make money from it, but why Microsoft don't care Windows Developer Community and more attention to foreign OS and platform?!!

**Comment by kvloover** [comment]
I agree that it's a bit weird that the 'native' OS is the last to receive the release. And we're already halfway through 2021.

I imagine it wouldn't be this bad if WSL worked without headaches to support running and sharing files into a linux container. Sadly this is also a bit too much hassle for what it's worth.

If I'd have to advise anybody investing in a development/production environment today, I'd push them towards a linux distro.

**Comment by amvin87** [comment]
Please refer the blog here which currently states that SQL Server containers on windows is suspended for now. https://techcommunity.microsoft.com/t5/sql-server/update-beta-program-for-sql-server-on-windows-container-is/ba-p/2516639 
If you want to use it for development and testing you can refer this sample dockerfile: https://github.com/microsoft/mssql-docker/blob/master/windows/mssql-server-windows-developer/dockerfile_1. Please do not use this for production workload because this is an unsupported configuration as informed earlier.
